cpp/OcvPersonDetection/plugin-files/descriptor/descriptor.json

```diff
@@ -65,6 +65,18 @@
           "defaultValue": "false"
         },
         {
+          "name": "MIN_GAP_BETWEEN_TRACKS",
+          "description": "When merge tracks is set to true, this defines the minimum number of distance in frames to allow between similar tracks before merging. A value of 2 or less indicates that adjacent frames will be merged.",
```

Comment:
Please rephrase:

"this defines the minimum distance in frames"

Please make this correction to all of the components.